<launch>

  <param name="imageTopic" value="/camera/rgb/image_rect_color" />
  <param name="cameraInfoTopic" value="/camera/rgb/camera_info" />
  <param name="parentNode" value="/talos/rgbd_rgb_optical_frame" />
  <param name="displayDebug" value=true />
  <param name="broadcastTf" value=true />

  <node pkg="agimus_vision" type="tracker_object" name="tag_tracker" />

</launch>